Tema-3/script5.js

```diff
@@ -0,0 +1,9 @@
+console.log ( "Exercitiul 5");
+
+const pretVanzare= [18000, 12000,  18000, 27300, 12000, 14000, 12000, 9000, 37000, 19900, 43000, 27400, 21500, 13590];
+ let profit= 0;
+
+for ( let i=0; i< pretVanzare.length; i++) {
+    profit= pretVanzare[i] - pretCumparareList[i];
```

Comment:
aici puteai declara direct bariabile ca si ```const profit = pretVanzare[i] - pretCumparareList[i]``` deoarece nu e folosita in afara listei

Reply:
Am observat ca mesajul e failed exercise 5, dar e corect, treci prin lista, si afisezi profitul pentur fiecare marca